### DISCUSSION TITLE
Is there a similar feature to rowview in v2?

### DISCUSSION THREAD

**Discussion by DanielGSoftware** [rappasoft/laravel-livewire-tables#829] [Q&A] [answered]
I'm currently upgrading our tables to V2, and we use a [custom row view ](https://rappasoft.com/docs/laravel-livewire-tables/v1/rows/custom-row-view) in many instances, because we think it's better than defining a lot of partials.

However, it seems like there's no similar option in V2, or am I wrong?

**Comment by paramwork** [comment]
Have you found the solution to this?

**Reply by DanielGSoftware** [reply]
> Have you found the solution to this?

Yes/no, there's no similar feature in v2.

**Reply by paramwork** [reply]
What are you currently using?

**Reply by DanielGSoftware** [reply]
Either `format`, `label` or `view`.

**Comment by imabug** [comment]
For most of my custom row views, I was able to replace them with format().  They were all relatively simple, so YMMV
https://rappasoft.com/docs/laravel-livewire-tables/v2/columns/available-methods#content-formatting

**Reply by DanielGSoftware** [reply]
Yep you're right. Format, label, or view can be used as an alternative, but it's not exactly the same as a rowView where you can write your whole row in 1 blade file :)

**Reply by paramwork** [reply]
My tables have more than 25 columns and I have around 50 tables. So, it is not time efficient for me to create a column view for each column and manage them. IMO rowView was easier to maintain.

**Reply by DanielGSoftware** [reply]
I agree, I've started a discussion about this and other changes in V2 that might no work as well as it did before. See [here](https://github.com/rappasoft/laravel-livewire-tables/discussions/888). Maybe we can try to implement rowView again

**Comment by rappasoft** [comment, marked as answer]
There is no row view in V2, I tried to make the column configuration as robust as possible so you wouldn't need it. The rows are just too complex now for adding your own, you'd lose a lot of the functionality.
